example-configs/yi-emacs-vty-static/Main.hs

```diff
@@ -1,7 +1,14 @@
+-- This is a configuration example, that requires manual rebuild (as opposed to dynamic ones, using
+-- Dyre, which you're, probably, actually want). Here's a building example with "stack":
```

Comment:
Ah, that's interesting details, I didn't know. So, just to recap, do you mean that it's possible to just copy the binary to other machine, and it'd work with disregard to availability of libs like microlens, ghc, etc?